cypress/endToEnd/i-can-update-a-registration.spec.ts

```diff
@@ -37,22 +38,58 @@ describe("As an account holder", () => {
 
     whenIHaveVisited(AccountPageURLs.accountHome);
     iCanSeeMyExistingRegistrationHexId(testRegistration.hexId);
-
-    whenIClickOnTheHexIdOfTheRegistrationIWantToUpdate(testRegistration.hexId);
+    iCanClickTheUpdateLinkToUpdateARegistration(testRegistration);
+    iCanAlsoClickTheHexIdOfTheRegistrationIWantToUpdate(testRegistration.hexId);
+    iCannotSeeAnAcceptAndSendButtonBecauseIHaveNotMadeAnyChanges();
     iCanUpdateTheDetailsOfMyExistingRegistration(testRegistration);
 
+    whenIClickTheButtonContaining("Accept and send");
+    thenTheUrlShouldContain(UpdatePageURLs.updateComplete);
+    iCanSeeAPageHeadingThatContains(
+      "Your beacon registration has been updated"
+    );
+
+    whenIClickTheButtonContaining("Return to your Account");
+    thenTheUrlShouldContain(AccountPageURLs.accountHome);
+
+    whenIClickTheHexIdOfTheRegistrationIWantToUpdate(testRegistration.hexId);
+    iCannotSeeAnAcceptAndSendButtonBecauseIHaveNotMadeAnyChanges();
     iCanViewTheUpdatedBeaconInformation(updatedRegistrationDetails);
     iCanViewTheUpdatedAdditionalBeaconInformation(updatedRegistrationDetails);
     iCanViewTheUpdatedUseInformation(updatedRegistrationDetails);
     iCanViewTheUpdatedOwnerInformation(updatedRegistrationDetails);
   });
 });
 
+const iCannotSeeAnAcceptAndSendButtonBecauseIHaveNotMadeAnyChanges = () => {
+  cy.get(`[role=button]:contains(accept and send)`).should("not.exist");
+};
+
+const iCanClickTheUpdateLinkToUpdateARegistration = (
+  registration: Registration
+) => {
+  cy.get("main")
+    .contains(registration.hexId)
+    .parent()
+    .parent()
+    .contains(/update/i)
+    .click();
+
+  iCanSeeMyBeaconInformation(registration);
+  iCanSeeAdditionalBeaconInformation(registration);
+  iCanSeeOwnerInformation(registration);
+  iCanSeeEmergencyContactInformation(registration);
+  iCanSeeUseInformation(registration);
+  theBackLinkGoesTo(AccountPageURLs.accountHome);
+  whenIClickBack();
+  thenTheUrlShouldContain(AccountPageURLs.accountHome);
+};
+
 const iCanViewTheUpdatedOwnerInformation = (
   draftRegistration: DraftRegistration
 ) => {
   whenIHaveVisited(AccountPageURLs.accountHome);
-  whenIClickOnTheHexIdOfTheRegistrationIWantToUpdate(testRegistration.hexId);
+  iCanAlsoClickTheHexIdOfTheRegistrationIWantToUpdate(testRegistration.hexId);
```

Comment:
```suggestion
  whenIClickOnTheHexIdOfTheRegistrationIUpdated(testRegistration.hexId);
  
```
Maybe this would be better here since the assertion is about viewing updated info